modules/ROOT/pages/database/database-connector-studio.adoc

```diff
@@ -0,0 +1,47 @@
+= Database Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+// Add transition
+
+To configure a connector in Anypoint Studio:
+
+. Install the connector.
+. Configure the connector.
+. Configure an input source for the connector.
+
+== Install the Connector
+
+. In Anypoint Studio, click the Anypoint Exchange icon in the Studio task bar.
+. In Anypoint Exchange, click *Login* and supply your Anypoint Platform username and password.
+. Search for the Database connector, click the card for the connector, and click *Add to project*.
+. Follow the prompts.
+
+== Configure the Connector
+
+No additional configuration is required. Optionally, you can configure global properties for this connector.
```

Comment:
Where are these? Why aren't they here? This isn't really useful if it doesn't point me to where the Global Properties are.

Reply:
They're a generic topic. Added a link.